models/vyos/bgp_address_family/vyos_bgp_address_family.yaml

```diff
@@ -0,0 +1,210 @@
+                type: str
+              metric:
+                description: Metric for redistributed routes.
+                type: int
+      neighbors:
+        description: BGP neighbor
+        type: dict
+        suboptions:
+          address:
+            description: BGP neighbor address (v4/v6).
+            type: str
+          afi:
+            description: BGP neighbor parameters.
+            type: str
+            choices: ['ipv4_unicast', 'ipv6_unicast']
+          allowas_in:
+            description: Number of occurrences of AS number.
+            type: int
+          as_override:
+            description:  AS for routes sent to this neighbor to be the local AS.
+            type: bool
+          attribute_unchanged:
+            description: BGP attributes are sent unchanged.
+            type: str
+            choices: ['as_path', 'med', 'next_hop']
+          capability:
+            description: Advertise capabilities to this neighbor.
+            type: dict
+            suboptions:
+              dynamic:
+                description: Advertise dynamic capability to this neighbor.
+                type: bool
+              orf:
+                description: Advertise ORF capability to this neighbor.
+                type: str
+                choices: ['send', 'receive']
+          default_originiate:
+            description: Send default route to this neighbor
+            type: str
+          distribute_list:
+            description:  Access-list to filter route updates to/from this neighbor.
+            type: list
+            elements: dict
+            suboptions:
+              action:
+                description:  Access-list to filter outgoing/incoming route updates to this neighbor
+                type: str
+                choices: ['export', 'import']
+              acl:
+                description: Acess-list number.
+                type: int
+          filter_list:
+            description: As-path-list to filter route updates to/from this neighbor.
+            type: list
+            elements: dict
+            suboptions:
+              action:
+                description: filter outgoing/incoming route updates
+                type: str
+                choices: ['export', 'import']
+              path_list:
+                description: As-path-list to filter
+                type: str
+          maximum_prefix:
+            description:  Maximum number of prefixes to accept from this neighbor
+               nexthop-self Nexthop for routes sent to this neighbor to be the local router.
+            type: int
+          nexthop_local:
+            description:  Nexthop attributes.
+            type: bool
+          nexthop_self:
+            description:  Nexthop for routes sent to this neighbor to be the local router.
+            type: bool
+          peer_group:
+            description:  IPv4 peer group for this peer
+            type: str
+          prefix_list:
```

Comment:
If we can have only two entries in config, one for export one for import, can we make this a dict with these two keys? The problem with keep this a list of dicts is, user can pass in more than two entries, which might lead to erroneous behaviour.
```
prefix_list:
  import: route_map_1
  export: route_map_2
```